### DISCUSSION TITLE
Cant find libcamerify

### DISCUSSION THREAD

**Discussion by HelpfulFix** [Motion-Project/motion#1620] [Q&A] [answered]
Hi There,

I am sure I have missed something but I can't find libcamerify anywhere. 

I've got a PI 3+ with an Imx327 camera that can only run in Bullseye (libcamera drivers only sadly). I want to create a security camera with it but it's all turning out to be a little bit tricky as support for libcamera is quite thin on the ground at this stage. 

Any help would be greatly appreciated.

**Comment by Mr-Dave** [comment, marked as answer]
At one point the libcamera developer [referenced](https://github.com/Motion-Project/motion/issues/1434#issuecomment-1060755121) it was in `libcamera-tools'  (It isn't part of Motion so we don't track where it may have gone to)

**Comment by nightcustard** [comment]
I've just installed the drivers for an ArduCam IMX519 using the instructions here: https://docs.arducam.com/Raspberry-Pi-Camera/Native-camera/Quick-Start-Guide/

libcamerify is installed as part of the libcamera package. 

Edit:  the IMX327 sensor is used in this ArduCam camera - https://www.arducam.com/product/presale-arducam-2mp-ultra-low-light-starvis-imx327-motorized-ir-cut-camera-for-raspberry-pi/  which suggests the above may point you in the right direction.

I've got quite a number of Pi NoIR cameras running on older versions of Motion.  This is my first attempt to try Motion with the libcamera support, so it looks like we're trying to do similar things - good luck!

**Reply by HelpfulFix** [reply]
Thanks for the replies! I found it in libcamera-tools. Although installing libcamera-tools added 300mb of packages (x11 etc). 

I wonder what the script depends on. I might try installing libcamera-tools with --ignore-depends and manually resolving dependencies.

**Comment by lupa18** [comment]
Hi ! I have bullseye (arm raspbian) and have installed libcamera-tools  0~git20200513+924778e-1.1 (old version). Any hint to get the libcamerify script installed on this system?

**Reply by simoncarrignon** [reply]
not sure if you find it but it is still in libcamera-tools ; so simply doing `sudo apt install libcamera-tools` should still work for bullseye too!
